### PR TITLE
MARP-463 Multilingualism for Website - landing page

### PR DESCRIPTION
**Pull request by phhung-axonivy**

### CODE REVIEW

File: src/main/java/com/axonivy/market/factory/ProductFactory.java [hunk 1]
```diff
@@ -68,6 +73,21 @@ public static Product mappingByMetaJSONFile(Product product, GHContent ghContent
...
 
+  private static MultilingualismValue mappingMultilingualismValueByMetaJSONFile(List<DisplayValue> list) {
+    MultilingualismValue value = new MultilingualismValue();
+    if (!CollectionUtils.isEmpty(list)) {
+      for (DisplayValue name : list) {
+        if (Language.EN.getValue().equalsIgnoreCase(name.getLocale())) {
+          value.setEn(name.getValue());
+        } else if (Language.DE.getValue().equalsIgnoreCase(name.getLocale())) {
+          value.setDe(name.getValue());
+        }
```
Comment (tutn-axonivy) [inline src/main/java/com/axonivy/market/factory/ProductFactory.java:84]: Handle for not matching any language case

File: src/main/java/com/axonivy/market/constants/GitHubConstants.java [hunk 2]
```diff
@@ -8,5 +8,6 @@ public class GitHubConstants {
 	public static final String AXONIVY_MARKET_ORGANIZATION_NAME = "axonivy-market";
 	public static final String AXONIVY_MARKETPLACE_REPO_NAME = "market";
 	public static final String AXONIVY_MARKETPLACE_PATH = "market";
+  public static final String DEFAULT_BRANCH = "feature/MARP-463-Multilingualism-for-Website";
```
Comment (tutn-axonivy) [inline src/main/java/com/axonivy/market/constants/GitHubConstants.java:11]: update default branch

File: src/main/java/com/axonivy/market/factory/ProductFactory.java (hunk 1 above)
Comment (phhung-axonivy) [reply src/main/java/com/axonivy/market/factory/ProductFactory.java:84]: We will check the structure of the meta JSON file on the repo market with the Jenkins pipeline, so I think it's not necessary to handle this case.

File: src/main/java/com/axonivy/market/constants/GitHubConstants.java (hunk 2 above)
Comment (phhung-axonivy) [reply src/main/java/com/axonivy/market/constants/GitHubConstants.java:11]: Ok, I will update it later when new meta JSON files are applied on the market repo.

**Review by tutn-axonivy — APPROVED** [review]
LGTM

**Pull request merged**